cpp/tests/io/json_tree.cpp

```diff
@@ -14,9 +14,11 @@
  * limitations under the License.
  */
 
+#include "cudf/detail/utilities/vector_factories.hpp"
```

Comment:
Move to the `cudf` block of includes.
```suggestion
#include <cudf/detail/utilities/vector_factories.hpp>
```